runtime/j9vm/java11vmi.c

```diff
@@ -1570,3 +1571,11 @@ JVM_AreNestMates(JNIEnv *env, jclass jClassOne, jclass jClassTwo)
 #endif /* J9VM_OPT_VALHALLA_NESTMATES */
 }
 #endif /* JAVA_SPEC_VERSION >= 11 */
+
+#if JAVA_SPEC_VERSION >= 15
+JNIEXPORT jlong JNICALL
+JVM_GetRandomSeedForCDSDump()
+{
+	return (jlong)J9UniqueBuildID;
```

Comment:
@keithc-ca is this called during an OpenJ9 run?  If we don't expect it to be (and why would it if it's CDS related), it would be better to assert here

Reply:
There are still some more changes required to openj9 to create a build against the openj9-staging branch of jdknext. My first observation was a link error referring to this function, but a closer look at how it's [used](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/master/src/java.base/share/classes/java/util/ImmutableCollections.java#L79) suggests it is required beyond support for CDS and that our implementation should just return zero. I'll update it to do that.

Reply:
Updated.